specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-07-09-preview/confidentialledger.json

```diff
@@ -0,0 +1,912 @@
+      "type": "string",
+      "description": "Object representing ProvisioningState for Confidential Ledger.",
+      "enum": [
+        "Unknown",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Creating",
+        "Deleting",
+        "Updating"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "RunningState": {
+      "type": "string",
+      "description": "Object representing RunningState for Confidential Ledger.",
+      "enum": [
+        "Active",
+        "Paused",
+        "Unknown",
+        "Pausing",
+        "Resuming"
+      ],
+      "x-ms-enum": {
+        "name": "RunningState",
+        "modelAsString": true
+      }
+    },
+    "LedgerRoleName": {
+      "description": "LedgerRole associated with the Security Principal of Ledger",
+      "type": "string",
+      "enum": [
+        "Reader",
+        "Contributor",
+        "Administrator"
+      ],
+      "x-ms-enum": {
+        "name": "LedgerRoleName",
+        "modelAsString": true
+      }
+    },
+    "LedgerSku": {
+      "description": "SKU associated with the ledger resource",
+      "type": "string",
+      "enum": [
+        "Standard",
+        "Basic",
+        "Unknown"
+      ],
+      "x-ms-enum": {
+        "name": "LedgerSku",
+        "modelAsString": true
+      }
+    },
+    "CertBasedSecurityPrincipal": {
+      "type": "object",
+      "description": "Cert based security principal with Ledger RoleName",
+      "properties": {
+        "cert": {
+          "description": "Public key of the user cert (.pem or .cer)",
+          "type": "string"
+        },
+        "ledgerRoleName": {
+          "$ref": "#/definitions/LedgerRoleName"
+        }
+      }
+    },
+    "AADBasedSecurityPrincipal": {
+      "type": "object",
+      "description": "AAD based security principal with associated Ledger RoleName",
+      "properties": {
+        "principalId": {
+          "description": "UUID/GUID based Principal Id of the Security Principal",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "UUID/GUID based Tenant Id of the Security Principal",
+          "type": "string"
+        },
+        "ledgerRoleName": {
+          "$ref": "#/definitions/LedgerRoleName"
+        }
+      }
+    },
+    "NodeCount": {
+      "type": "integer",
+      "format": "int32",
+      "description": "Number of CCF nodes in the Managed CCF.",
+      "title": "NodeCount",
+      "default": 3
+    },
+    "CustomizationData": {
+      "type": "object",
+      "description": "Object representing the different fields of customization data.",
+      "title": "CustomizationData",
+      "properties": {
+        "hostLevel": {
+          "type": "string",
+          "readOnly": false,
+          "description": "CCF Property for the logging level for the untrusted host: Trace, Debug, Info, Fail, Fatal."
+        },
+        "maxBodySize": {
```

Comment:
I'd suggest this to be `maxBodySizeInMB` or something along those lines to specify that this value is on MB. 
I know there is a description, but it'd ideal to make a more specific field

Reply:
That makes sense. addressed